docs/DeveloperGuide.md

```diff
@@ -548,10 +547,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | nurse                       | view my schedule on a particular day                          | make plans for that day in advance                                                                                                               |
 | `* * *`  | nurse                       | search a task up by its name                                  | quickly get the details of a task                                                                                                                |
 | `* * *`  | careless nurse              | undo my recent actions                                        | revert the database to previous changes in case I make a mistake                                                                                 |
+| `* * *`  | careless nurse              | redo my recent actions                                        | re-execute the actions that I have `undo`(ne) recently                                                                                           |
```

Comment:
Hmm I feel that 'redo my recent actions' is not very accurate. Perhaps a better way to phrase it would be: As a careless nurse, I can redo previously undone commands, so that I can reverse the previous `undo` commands and re-execute the actions.